src/main/java/me/truemb/rentit/api/AdvancedChestsUtils.java

```diff
@@ -47,19 +46,12 @@ public Map<String, List<String>> getChestsInArea(BlockVector3 min, BlockVector3
                     AdvancedChest chest = AdvancedChestsAPI.getChestManager().getAdvancedChest(location);
 
                     if (chest != null) {
-                        /*
-                         * If there is an advanced chest in the given location, we recover the location
```

Comment:
I spoke with the Advanced Chests dev about this issue with the yaw and pitch back in the day. It's now supported natively by their `LocationUtils.serializeLoc(location)` function so no need for the workaround anymore.